setup.py

```diff
@@ -16,14 +16,17 @@
     "PyYAML>=4.2b1",
 ]
 
+# docker is needed to use moto 3.0.0 and above
+# see https://github.com/spulec/moto/issues/3567#issuecomment-947047013
 dev_requires = [
     "black==22.3.0",
+    "docker>=5.0.0",
     "flake8>=3.3.0",
     "isort==4.3.21",
     "pytest>=3.6",
     "pytest-cov>=2.5.1",
     "pip-tools>=5.3.1",
-    "moto==1.3.14",
+    "moto>=3.0.0",
```

Comment:
Let's use the extra requirements provided by moto
```suggestion
    "moto[cloudformation,s3]>=3.0.0",
```

Reply:
done in [12062ed](https://github.com/Skyscanner/cfripper/pull/219/commits/12062ed53e819209ee3c996a7134aadf0738a5ec)